disco/disco.go

```diff
@@ -118,6 +118,8 @@ type Ping struct {
 	// netmap data to reduce the discokey:nodekey relation from 1:N to
 	// 1:1.
 	NodeKey key.NodePublic
+
+	Dst netip.AddrPort
```

Comment:
This needs to be documented.

Reply:
> This needs to be documented.
        // Dst is the destination IP/Port of the ping message, it's receiving
        // node's public facing address relative to sending node. The receiving
        // node will add this to its local endpoint list upon receiving the
        // ping message.

Will this work? If the comment is OK I'll add to my PR.